### ISSUE TITLE
Unable to sync to Azure Git Repo

### ISSUE THREAD

**Issue by mzaferyahsi** [xpipe-io/xpipe#308]
Hi,

I'd like to synchronize to Azure DevOps Git Repo, but I'm getting the following error with following settings.

```
Repo: git@ssh.dev.azure.com:v3/{{ORG}}/{{PROJECT}}xpipe
SSH Identity: SSH-Agent
```

The error message is;
```
remote: TF401019: The Git repository with name or identifier xpipe.git does not exist or you do not have permissions for the operation you are attempting.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

When I clone the repo via terminal, I can clone it via the exact url `git@ssh.dev.azure.com:v3/{{ORG}}/{{PROJECT}}xpipe` but cannot clone it via `git@ssh.dev.azure.com:v3/{{ORG}}/{{PROJECT}}xpipe`.

Am I wrong to assume that `.git` is being added by XPipe to the git url? If this is the case, it prevents me to use Azure DevOps since it doesn't support .git extension at the repo url.

**Comment by crschnick** [comment]
Yeah it adds `.git` since I assumed that this was supported everywhere.

I could easily remove this behavior for the next release tomorrow, but that will probably break a few other cases where a `.git` is required. Sadly many git repository providers are inconsistent with the behavior on how the URLs should look like.

I'm not sure how to handle this the best way.

**Comment by crschnick** [comment]
Alright having looked at it, I think I will just remove all git url transformations as they can never cover all cases and will lead to problems like this.

In conjunction, I improved the error messages when a remote git connection fails to hopefully help users in cases where they either are missing the suffix or have added when it's not supported like in this case.

This will be released tomorrow in 10.2. If you want, you can try out an early version at https://github.com/xpipe-io/xpipe-ptb and see whether it can connect. To enable git sync in the PTB build, you first have to enable developer mode under system settings and restart.

**Comment by crschnick** [comment]
This should now be fixed in https://github.com/xpipe-io/xpipe/releases/tag/10.2

**Comment by mzaferyahsi** [comment]
Indeed, it's fixed now! Thanks!
